cbmc/proofs/IotMqtt_DeserializeSuback/Makefile

```diff
@@ -21,10 +21,9 @@ DEF += -DBUFFER_SIZE=$(BUFFER_SIZE)
 # proof and assume their implementation is correct
 ABSTRACTIONS += --remove-function-body IotLog_Generic
 
-# These defines have the effect of removing RemoveAllMatches from the
+# This defines have the effect of removing RemoveAllMatches from the
```

Comment:
"This define has" if you really want to make this change...

Reply:
Done!